spec/dmv_data_service_spec.rb

```diff
@@ -25,6 +25,12 @@
     end
   end
 
+  describe '#ny_registrations' do
+  it 'can load ny registration data' do
+    expect(@dds.ny_registrations.size).to be_an(Integer)
```

Comment:
There is a better test here - what can you test about the first element of this dataset? Validate the data is the shape you expect it to be